examples/test_database.py/TestDatabaseReader.ipynb

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
can we put TestDatabaseReader under the `data_connectors` folder in examples? (and name it DatabaseReaderDemo.ipynb)